crates/aptos/src/common/utils.rs

```diff
@@ -97,19 +103,22 @@ pub fn cli_build_information() -> BTreeMap<String, String> {
     build_information!()
 }
 
+pub static CLI_TELEMETRY: Lazy<Arc<Mutex<BTreeMap<String, String>>>> =
+    Lazy::new(|| Arc::new(Mutex::new(build_information!())));
+
 /// Sends a telemetry event about the CLI build, command and result
 async fn send_telemetry_event(
     command: &str,
     latency: Duration,
     success: bool,
     error: Option<&str>,
 ) {
-    // Collect the build information
-    let build_information = cli_build_information();
+    let mut telemetry = CLI_TELEMETRY.lock().unwrap().clone();
+    collect_limited_system_information(&mut telemetry);
```

Comment:
I believe doing such things in the end would delay process exit and could create bad experience. Have we measured how long this takes? Ideally, we should collect these info in a parallel thread/task. 

One reason we haven't migrated to telemetry service for CLI is that it noticeably increased the process exit time, because we were doing 2 RTTs to post to telemetry service.

Reply:
I'll give it a test with a compiled binary